components/squad/squad_custom.lua

```diff
@@ -0,0 +1,38 @@
+local Squad = require('Module:VoganRL/Squad')
+local SquadRow = require('Module:VoganRL/Squad/Row')
+local Json = require('Module:Json')
+
+
+local CustomSquad = {}
+
+function CustomSquad.run(frame)
+	local squad = Squad()
+	squad:init(frame):title():header()
+
+	local args = squad.args
+
+	local index = 1
+	while args['p' .. index] ~= nil do
+		local player = Json.parseIfString(args['p' .. index])
```

Comment:
```suggestion
	while args['p' .. index] or args[index] do
		local player = Json.parseIfString(args['p' .. index] or args[index])
```
If we want to support both name and unnamed parameters (optional)

Reply:
Done